### DISCUSSION TITLE
关于通过脚本自动获取/刷新token

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [wanghongenpin/proxypin#345] [Q&A]
不知是不是可以通过自定义脚本来自动获取授权token，一般在开发中如果token过期，可以实现静默刷新，这样只需要定义好用户名密码。就不需要在手工的输入token了，检测到403会自动获取新的token。这个在apifox上可以实现。是否支持或者有示例？

**Comment by wanghongenpin** [comment]
可以用脚本做
